testsuite/integration-tests/pom.xml

```diff
@@ -348,6 +348,37 @@
                 </excludes>
             </testResource>
         </testResources>
+        <plugins>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <id>default-test</id>
+                            <configuration>
+                                <includes>
+                                    <include>**/Test*.java</include>
+                                </includes>
+                                <excludes>
+                                     <!-- Tests requires excluding JsonBindingProvider-->
+                                     <exclude>**/JsonBindingTest.java</exclude>
+                                </excludes>
+                                <classpathDependencyExcludes>
+                                    <classpathDependencyExcludes>org.jboss.resteasy:resteasy-json-binding-provider</classpathDependencyExcludes>
```

Comment:
mmh, I'm not really sure about this. Does it mean that the new json-b provider is going to break the behavior of most apps out there when they upgrade to latest resteasy? If that's really the case (is this a consequence of the @Priority you added?), we should evaluate different solutions (like not having direct and transitive dependencies to the new module by default and adding it explicitly in the json-b test). Do we really have such a non backward compatibility?